tbot_contrib/swupdate/__init__.py

```diff
@@ -0,0 +1,26 @@
+import tbot
+import pathlib
+from tbot.tc import shell
+from tbot.machine.linux import Path
+
+
+@tbot.testcase
+def swupdateweb_lab(path_to_swu_file: Path, ip: str) -> None:
+    with tbot.acquire_lab() as lh, tbot.acquire_local() as lo:
+        """copy the files from the local host to the lab host and the runs the swupdate_script.py on the lab host"""
+        p = Path(lo, pathlib.Path.cwd())
+        script_path = p / "swupdate_script.py"
+        script_on_labhost = lh.workdir / "swupdate_script.py"
+        swu_on_labhost = lh.workdir / "image.swu"
+        tbot.log.message("aqui")
+        shell.copy(script_path, script_on_labhost)
+        shell.copy(path_to_swu_file, swu_on_labhost)
+        lh.exec0("python3", script_on_labhost, swu_on_labhost, ip)
+
+
+@tbot.testcase
+def swupdateweb_local(path_to_swu_file: Path, ip: str) -> None:
```

Comment:
Ideally, we want a single testcase that works regardless of whether the lab is local or remote.  What about something like this:

```python
import tbot
from tbot.machine import linux
from tbot.tc import shell

@tbot.testcase
@tbot.with_lab
def swupdate_update_web(lh: linux.Lab, swu_file: linux.Path, target_ip: str) -> None:
    with tbot.acquire_local() as lo:  # Needed for the script
        script_path = lh.workdir / "tbot_swupdate_web.py"
        script_source = linux.Path(lo, __file__).parent / "swupdate_script.py"
        shell.copy(script_source, script_path)

        ...
```

`shell.copy()` is smart enough to work in all cases (I hope).  So even if the lab-host is the localhost, this test should still work.